manager/apis/app/v1alpha1/ifdetails.go

```diff
@@ -18,6 +18,7 @@ const (
 	CSV     DataFormatType = "csv"
 	JSON    DataFormatType = "json"
 	AVRO    DataFormatType = "avro"
+	ORC     DataFormatType = "orc"
 	Binary  DataFormatType = "binary"
 	Arrow   DataFormatType = "arrow"
 )
```

Comment:
For consistency, please add the new format in https://github.com/IBM/the-mesh-for-data/blob/6c083e8b7427fbd775d81fea3b9c52f9702da4a1/samples/gui/front-end/src/formats/InputDataTable.js#L4